bundles/org.openhab.binding.comfoair/README.md

```diff
@@ -4,6 +4,8 @@ This binding allows to monitor and control Zehnder ComfoAir serial controlled ve
 Though the binding is developed based on the protocol description for Zehnder ComfoAir devices it should also work for mostly identical systems from different manufacturers, like StorkAir WHR930, Wernig G90-380 and Paul Santos 370 DC.
 It was also successfully tested on a Wernig G90-160.
 
+!!! This binding does not work on the ComfoAir Q-series (Q350 / Q450). These ventilation units use a CAN bus for communication, see the zcan or ComfoConnect projects for integration !!!
```

Comment:
I agree that this might be worth a note, but I don't think that the high number of exclamation marks adds any value here. Also, I would refrain from mentioning external projects here since I don't think that these are helpful without any additional instructions (but since I'm no maintainer, I don't know if this is common).

```suggestion
**NOTE:** This binding does not work with the ComfoAir Q-series (e.g. Q350 or Q450). These ventilation units use a CAN bus for communication and a different protocol.
```